src/Silex/WebTestCase.php

```diff
@@ -53,14 +55,18 @@ public function setUp(): void
      *
      * @param array $server Server parameters
      *
-     * @return Client A Client instance
+     * @return AbstractBrowser|Client A Client instance
      */
     public function createClient(array $server = array())
     {
         if (!class_exists('Symfony\Component\BrowserKit\Client') && !class_exists('Symfony\Component\BrowserKit\AbstractBrowser')) {
             throw new LogicException('Component "symfony/browser-kit" is required by WebTestCase.'.PHP_EOL.'Run composer require symfony/browser-kit');
         }
 
-        return new Client($this->app, $server);
+        if (class_exists(Client::class)) {
```

Comment:
same point, seems that we should be developing against symfony 5 (HttpKernelBrowser was added so I am guessing that this is the new class) and drop to 4 if it doesnt exist